allauth/account/forms.py

```diff
@@ -363,7 +355,6 @@ class AddEmailForm(UserForm):
 
     def clean_email(self):
         value = self.cleaned_data["email"]
-        value = get_adapter().clean_email(value)
```

Comment:
We cannot remove this one. There are use cases where people override the `clean_email` such that you can only add @theirowncompany.com based email addresses. With this change, any email address can be added.